hawc/services/epa/hero.py

```diff
@@ -90,24 +124,47 @@ def get_content(self):
             self.failures = []
             return dict(success=self.content, failure=self.failures)
 
+        if settings.HAWC_FEATURES.ENABLE_NEW_HERO:
```

Comment:
I removed the key validation check if it fails, we'll get an error, but we send fewer requests to HERO